Dockerfile

```
@@ -0,0 +1,53 @@
+# Use the official Node.js LTS image as the base image
+FROM node:lts
+
+# Create and change to the app directory
+WORKDIR /usr/src/app
+
+# Copy package.json and package-lock.json to the app directory
+COPY package*.json ./
+
+# Install app dependencies
+RUN npm ci
```

Anticipating Rudolf's comment 😄
```suggestion
RUN npm clean-install
```

---

👍